x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/type/DataType.java

```diff
@@ -205,13 +202,28 @@
      */
     public final boolean defaultDocValues;
 
-    DataType(SQLType sqlType, int size, int defaultPrecision, int displaySize, boolean isInteger,
-            boolean isRational, boolean defaultDocValues) {
+    DataType(
+        SQLType sqlType,
+        int size,
+        int defaultPrecision,
+        int displaySize,
+        boolean isInteger,
+        boolean isRational,
+        boolean defaultDocValues
+    ) {
         this(null, sqlType, size, defaultPrecision, displaySize, isInteger, isRational, defaultDocValues);
     }
 
-    DataType(String esType, SQLType sqlType, int size, int defaultPrecision, int displaySize, boolean isInteger,
-             boolean isRational, boolean defaultDocValues) {
+    DataType(
```

Comment:
As it was before the change this PR proposes, constructors in SQL were consistent in style (hopefully, all of them). The proposed change, though, diverts quite a lot from the rest of code (at least in the SQL code base). Is there a plan to change this everywhere, or the change will be only in this class?

Reply:
The plan is format everything just before the `8.x` branch is taken, as well as `7.x` branch, so all files will have a consistent style.

Reply:
I'm surprised as well by the approach taken when wrapping constructors which I find inconsistent vs regular methods.
Either for method declarations we put each parameter on its own line or we do typical wrapping (break the line and continue onto the next one).

The change above is extreme - a method can wrap parameters on 1-2 lines but constructors (which are still methods) grew to 9 (!) lines. Also that's the only constructor that has been changed, the rest of the constructors inside SQL were not touched (and I'm happy that's the case).